src/jit/compiler.h

```diff
@@ -6625,8 +6626,14 @@ class Compiler
     {
         return codeGen->doDoubleAlign();
     }
-    DWORD getCanDoubleAlign(); // Defined & used only by RegAlloc
-#endif                         // DOUBLE_ALIGN
+    DWORD getCanDoubleAlign();
+    bool shouldDoubleAlign(unsigned refCntStk,
+                           unsigned refCntReg,
+                           unsigned refCntWtdReg,
+                           unsigned refCntStkParam,
+                           unsigned refCntWtdStkDbl);
```

Comment:
I got confused it is not under ifdef DEBUG
It is under ifdef DOUBLE_ALIGN which is fine.